cmd/test.go

```diff
@@ -448,17 +448,6 @@ func (t *Test) GetCmd() *cobra.Command {
 					IgnoreOrdering:     ignoreOrdering,
 					PassthroughHosts:   passThroughHosts,
 				}, enableTele)
-
-				fileExist := utils.CheckFileExists(path)
```

Comment:
You have still not added the changes to this file

Reply:
Incase it wasn't clear, earlier we were checking if the file exists here and if it did we would use chmod to change the permissions of the file. Now, it has been refactored so that the permissions are set appropriately right when the testReport file is being generated